### PR TITLE
Add Contributing file

### PR DESCRIPTION
**Pull request by maclover7**
cc @schneems

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -433,36 +434,16 @@ def self.call(input)
...
-
 ## License
-
-Copyright &copy; 2014 Sam Stephenson <<sstephenson@gmail.com>>
-
-Copyright &copy; 2014 Joshua Peek <<josh@joshpeek.com>>
-
-Sprockets is distributed under an MIT-style license. See LICENSE for
-details.
+Sprockets is released under the [MIT License](MIT-LICENSE).
```
Comment (rafaelfranca) [inline README.md]: Can we keep the Copyright messages?

File: README.md [hunk 2]
```diff
@@ -433,36 +434,16 @@ def self.call(input)
...
-submit a pull request.
+See [CONTRIBUTING](CONTRIBUTING.md).
 
 ### Version History
 
 Please see the [CHANGELOG](https://github.com/rails/sprockets/tree/master/CHANGELOG.md)
 
-## Build Status
-
-[![Build Status](https://travis-ci.org/rails/sprockets.svg?branch=master)](https://travis-ci.org/rails/sprockets)
```
Comment (rafaelfranca) [inline README.md]: Please keep the build status section here

**Comment by rafaelfranca** [comment]
Awesome! Thank you

File: README.md (hunk 1 above)
Comment (maclover7) [inline README.md]: Same copyright messages are available in the license file itself

File: README.md (hunk 2 above)
Comment (maclover7) [inline README.md]: Moved Travis badge to top of Readme - no need to show it twice?

Comment (rafaelfranca) [inline README.md]: Actually we don't want it on the top of the Readme, it is something only useful for developers so there is no need to give the importance we are giving with it on the top.

Comment (rafaelfranca) [inline README.md]: BTW, sorry for not being clear on the comment.

**Comment by robin850** [comment]
Nice work! Sorry if I'm misunderstanding something but what's the purpose of the ["Update CHANGELOG Again"](https://github.com/rails/sprockets/pull/105/files#diff-6a3371457528722a734f3c51d9238c13R91) part please ? I don't think it's needed as it's already explained [here](https://github.com/rails/sprockets/pull/105/files#diff-6a3371457528722a734f3c51d9238c13R51).

File: README.md (hunk 1 above)
Comment (schneems) [inline README.md]: Seems fine to have it in the license.

File: README.md (hunk 2 above)
Comment (schneems) [inline README.md]: Can you move the travis badge to the `##  contributing to sprockets` section and I think this is good to merge. Thanks for your work :heart:

**Pull request merged**